src/runtime-pmix.c

```diff
@@ -106,13 +106,43 @@ shmem_runtime_get_rank(void)
     return myproc.rank;
 }
 
-
 int
 shmem_runtime_get_size(void)
 {
     return size;
 }
 
+int
+shmem_runtime_get_local_rank(int pe)
+{
+#ifdef USE_ON_NODE_COMMS
+    pmix_status_t rc;
+    pmix_proc_t proc;
+    pmix_value_t *val;
+    uint16_t local_rank;
+
+    (void)strncpy(proc.nspace, myproc.nspace, PMIX_MAX_NSLEN);
```

Comment:
Why the cast to `(void)`?

Reply:
This function is going to be on a latency sensitive path.  We might want to pull the data out of PMIx into a table to reduce the overhead of this query.

Reply:
The cast was done [further up in `shmem_runtime_init`](https://github.com/Sandia-OpenSHMEM/SOS/blob/master/src/runtime-pmix.c#L51), but I don't see why... I'll remove all instances of this unless you say otherwise.

Sounds good with the table.